### DISCUSSION TITLE
[Enhancement] "Additional notes" section which isn't a step, but extra information about a recipe

### DISCUSSION THREAD

**Discussion by shayaknyc** [cooklang/spec#81] [Ideas]
Some recipes have additional information that could be noted about the preparation, but are not a discrete step. For example, I was working on a cookie recipe which has a filling. I wanted to include some notes that if you wish to use certain types of fillings, then one would need to use egg whites to bind the corners of the cookie instead of water, to prevent it from exploding during the baking process. The note appears as a last step (since it's at the end of the document) and not sure if there's a way to delineate this further.

I thought I might be able to use markdown syntax, but it's completely ignored and displays literally. Would be great to have something like this, or if someone knows something I don't, please point me in the right direction.

Thank you!

**Comment by dubadub** [comment]
Agree, this would be useful. We haven't decided on syntax yet. Here is a relevant discussion https://github.com/cooklang/spec/discussions/46.

(Originally posted in CookCLI issues).

**Comment by DanMacKay1961** [comment]
Agreed. I created a syntax for >>note: 

You can see it in use in the float-right boxes here: https://bonmot.ca/~daniel/Recipes/Baked_Beans_Pressure_Cooker

**Comment by dubadub** [comment]
This is now part of the main specification, closing the discussion.
